### DISCUSSION TITLE
Pybullet: Attaching a deformable URDF object to a standard URDF object

### DISCUSSION THREAD

**Discussion by tsreyas** [bulletphysics/bullet3#4398] [Q&A]
Hi, I've been experimenting with Pybullet for a uni project, and would like to know: if I have a standard URDF object defined with links, eg.

```
<robot name = x>
    <link name = "baseLink">
    ...
    </link>
</robot>
```

 is it possible to attach a deformable URDF (such as the one in the deformable_torus.py example) to it? If yes, how would I go about it?

**Comment by sallaimartin** [comment]
Hi, any update on this one? I am interested in doing the same.
